src/index.js

```diff
@@ -4,10 +4,11 @@ import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
+
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+      <App />
```

Comment:
remove unnecessary whitespaces

Reply:
sure ,removed some spaces ,please check once if anything required to change